Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/UpdateTextAccessibilityController.java

```diff
@@ -0,0 +1,30 @@
+package edu.asu.spring.quadriga.web.workspace;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.spring.quadriga.domain.workspace.ITextFile;
+import edu.asu.spring.quadriga.exceptions.FileStorageException;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.textfile.ITextFileManager;
+
+@Controller
+public class UpdateTextAccessibilityController {
+
+    @Autowired
+    private ITextFileManager tfManager;
+
+    @RequestMapping(value = "/auth/workbench/workspace/{workspaceId}/{textId}/updateAccessibility", method = RequestMethod.GET)
+    public String updateTextFileAccessibility(Model model, @PathVariable("textId") String textId,
```

Comment:
don't assume htere are only two types of accessibility. it's an enum field, so we can add an infinite number of options.